components/library/openssl/openssl-3.1/Makefile

```diff
@@ -0,0 +1,140 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+# Copyright (c) 2011, 2020, Oracle and/or its affiliates.
+#
+
+BUILD_BITS= 32_and_64
+include ../../../../make-rules/shared-macros.mk
+
+COMPONENT_VERSION_SHORT= 3.1
+COMPONENT_NAME=	openssl-$(COMPONENT_VERSION_SHORT)
+
+# When new version of OpenSSL comes in, you must update both COMPONENT_VERSION
+# and IPS_COMPONENT_VERSION.
```

Comment:
This comment deserves update, or removal.

Reply:
Removed